docs/docs/creating-a-source-plugin.md

```diff
@@ -432,7 +432,7 @@ If possible, the proactive listener approach creates the best experience if exis
 The code to support this behavior looks like this:
```

Comment:
```suggestion
Here's some pseudo code that shows this behavior:
```

Reply:
I've committed this change. Thanks for the suggestion!